packages/typescript-estree/tests/ast-alignment/utils.ts

```diff
@@ -274,6 +274,7 @@ export function removeLocationDataAndSourceTypeFromProgramNode(
   if (ignoreSourceType) {
     delete ast.sourceType;
   }
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Will be fixed by https://github.com/typescript-eslint/typescript-eslint/pull/3471

Reply:
but that's this PR!?

Reply:
![Spiderman pointing at spiderman meme](https://user-images.githubusercontent.com/3335181/122314107-89395280-cee5-11eb-821b-f37bebe9f91b.png)

#3498